library/oneview_storage_volume_attachment_facts.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/python
+###
+# Copyright (2016) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from ansible.module_utils.basic import *
+from hpOneView.oneview_client import OneViewClient
+from hpOneView.common import transform_list_to_dict
+
+DOCUMENTATION = '''
+---
+module: oneview_storage_attachment_facts
```

Comment:
The module name is different to file name.
oneview_storage_volume_attachment_facts